admin/fleetcommander/libvirtcontroller.py

```diff
@@ -236,7 +236,7 @@ def _generate_new_domain_xml(self, xmldata):
         root.find('uuid').text = newuuid
         # Change domain name
         name = root.find('name').text
-        root.find('name').text = '%s-fc-%s' % (name, newuuid)
+        root.find('name').text = '%s-fc-%s' % (name, newuuid[:8])
```

Comment:
you're capping the uuid to 8 characters, however 'name' can be of any arbitrary length so I don think this ensures this problem will not show up again in the future, why don't we set a name that has a fixed length, like "fc-%s" % newuuid?